backend/tests/integration/records/test_utils.py

```diff
@@ -56,3 +57,57 @@ def test_get_pids_for_one_pid(inspire_app):
     expected_recid = str(rec["control_number"])
     recid = get_pid_for_pid("cnum", rec["cnum"], "recid")
     assert expected_recid == recid
+
+
+def test_get_parent_record(inspire_app):
+    parent_record = create_record("lit")
+    data = {
+        "publication_info": [
+            {
+                "parent_record": {
+                    "$ref": f"http://localhost:5000/literature/{parent_record['control_number']}"
+                }
+            }
+        ]
+    }
+    rec = create_record("lit", data=data)
+    extracted_parent_record = get_parent_record(rec)
+    assert dict(extracted_parent_record) == dict(parent_record)
```

Comment:
why we need `dict`? :)

Reply:
Because sometimes I don't think what I'm doing ;) :+1: